build/generate-release-list.js

```diff
@@ -0,0 +1,21 @@
+const octokit = require('@octokit/rest')();
```

Comment:
How/when is this being executed? I don't see any references to it in the `package.json`. Does this tool belong elsewhere?

Reply:
It's referenced in https://github.com/mapbox/gl-internal/wiki/Javascript-release-process#publish-release-list